Makefile

```diff
@@ -49,12 +51,12 @@ build-deb-all: prepare-source
 	dpkg-buildpackage -i -I -rfakeroot
 
 srpm: source
-	rpmbuild --define '_topdir %{getenv:PWD}' \
-		 -bs python-aexpect.spec
+	if test ! -d BUILD/SRPM; then mkdir -p BUILD/SRPM; fi
+	mock --old-chroot -r $(MOCK_CONFIG) --resultdir BUILD/SRPM -D "commit $(COMMIT)" -D "commit_date $(COMMIT_DATE)" --buildsrpm --spec python-aexpect.spec --sources SOURCES
 
-rpm: source
-	rpmbuild --define '_topdir %{getenv:PWD}' \
-		 -ba python-aexpect.spec
+rpm: srpm
+	if test ! -d BUILD/RPM; then mkdir -p BUILD/RPM; fi
+	mock --old-chroot -r $(MOCK_CONFIG) --resultdir BUILD/RPM -D "commit $(COMMIT)" -D "commit_date $(COMMIT_DATE)" --rebuild BUILD/SRPM/python-aexpect-$(VERSION)-*.src.rpm
```

Comment:
Are you sure the `--old-chroot` is necessary? I tried it without it on F25 and it worked well. Perhaps it's time to remove this option.